pkg/client/client_venconn.go

```diff
@@ -57,11 +55,6 @@ type VenConnClient struct {
 // `restcfg` is not mutated. `trustedCAs` is only used for connecting to Venafi
 // Cloud and Vault and can be left nil.
 func NewVenConnClient(restcfg *rest.Config, agentMetadata *api.AgentMetadata, installNS, venConnName, venConnNS string, trustedCAs *x509.CertPool, disableCompression bool) (*VenConnClient, error) {
-	// TODO(mael): The rest of the codebase uses the standard "log" package,
-	// venafi-connection-lib uses "go-logr/logr", and client-go uses "klog". We
-	// should standardize on one of them, probably "slog".
-	ctrlruntimelog.SetLogger(logr.Logger{})
```

Comment:
This was from https://github.com/jetstack/jetstack-secure/pull/552:
 * https://github.com/jetstack/jetstack-secure/pull/552